run/GCHP/geoschem_config.yml.templates/geoschem_config.yml.CO2

```diff
@@ -15,6 +15,13 @@ simulation:
   species_metadata_output_file: OutputDir/geoschem_species_metadata.yml
   debug_printout: false
 
+#============================================================================
+# Timesteps settings
+#============================================================================
+timesteps:
+  transport_timestep_in_s: ${RUNDIR_TRANSPORT_TS}
+  chemistry_timestep_in_s: ${RUNDIR_CHEMISTRY_TS}
```

Comment:
These timesteps are actually set in setCommonRunSettings.sh since they are resolution-dependent for GCHP. Maybe there should be a note about that in the header for this section.

Reply:
@lizziel, I can add that note.

Reply:
Now added in https://github.com/geoschem/geos-chem/pull/1589/commits/08f1537ea9e68d4e293b7db95dc538321d7f196c.